spec/support/my_job.rb

```diff
@@ -0,0 +1,13 @@
+class MyJob
+  include Sidekiq::Job
+  sidekiq_options retry: 0
+
+  def perform(arg)
+    if arg == 'fail'
+      Sidekiq.logger.error('Forced failure for testing')
```

Comment:
But that will actually be a successful job, no? You would need to raise an error to blow it up